src/component-library/CTA/CTA.tsx

```diff
@@ -1,44 +1,49 @@
 import { ButtonHTMLAttributes, forwardRef } from 'react';
 
-import { CTAVariants, Sizes } from '../utils/prop-types';
-import { OutlinedCTA, PrimaryCTA, SecondaryCTA } from './CTA.style';
+import { LoadingSpinner } from '../LoadingSpinner';
+import { Sizes, Variants } from '../utils/prop-types';
+import { BaseCTA, BaseCTAProps } from './BaseCTA';
+import { LoadingWrapper } from './CTA.style';
+
+const loadingSizes: Record<Sizes, number> = {
```

Comment:
Any reason these aren't in the theme config?

Reply:
Because our LoadingSpinner does not expect a size but `diameter`. I agree this should go into the theme, but Im afraid the PR would go out of scope. Should I create a ticket for this?

Reply:
Yes create a ticket, but you're right it's out of scope.